grants/casper_gas_station/m6.md

```diff
@@ -0,0 +1,14 @@
+---
+Milestone: "M6",
+Status: "To do",
+---
+<!--lang:en--> 
+### Description
+
+- Implement Landing Page
+- Implement the development documentation with example
+
+### Acceptance Criteria
+
+- dApps builder can easily to integrate with Casper Gas Station with the clear and clean documentation
```

Comment:
Typo error on the sentence - please remove "to" in "dApps builder can easily to integrate with Casper Gas Station with the clear and clean documentation"